docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
-# Man Page
+# Man pages
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+pbench-clear-results, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [-C][--help]`
+
+**DESCRIPTION**  
+This command clears the results directory from /var/lib/pbench-agent directory
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+pbench-clear-tools, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [-C][-n][-g][-r][--help]
+`
+
+**DESCRIPTION**  
+Clear all tools which are registered and can filter by name of the group
+
+**OPTIONS**  
+-C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-n, --name,--names STR  
+Clear only the <tool-name> tool.
+
+-g, --group,--groups STR  
+Clear the tools in the <group-name> group. If no group is specified, the 'default' group is assumed.
+
+-r, --remote, --remotes STR  
+Clear the tool(s) only on the specified remote(s). Multiple remotes may be specified as a comma-separated list. If no remote is specified, all remotes are cleared
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-copy-results
+
+---
+
+**NAME**  
+pbench-copy-results, Copies result tarball to the server
+
+**SYNOPSIS**  
+`pbench-copy-results [--help] --user=<user> [--controller=<controller>] [--prefix=<path>][--xz-single-threaded] [--show-server]
+`
+
+**DESCRIPTION**  
+It keeps the copy of tarball on the pbench-agent and pushes the other to pbench-server
+
+**OPTIONS**  
+--user  
+This option value is required if not provided by the
+'PBENCH_USER' environment variable; otherwise, the value provided
+on the command line will override any value provided by the
+environment.
+
+--controller  
+This option may be used to override the value
+provided by the 'PBENCH_CONTROLLER' environment variable; if
+neither value is available, the result of 'hostname -f' is used.
+(If no value is available, the command will exit with an error.)
+
+--prefix  
+This option allows the user to specify an optional
+directory-path hierarchy to be used when displaying the result
+tar balls on the pbench server.
```

Comment:
Maybe "result files" rather than "result tar balls"? The prefix is used in the 0.69 `results` tree, which is ultimately symlinks into the unpacked `incoming` directories. Also, "0.69 Pbench Server" might be a worthwhile clarification here. The 1.0 Pbench Server has no equivalent concept since we don't have external links into a permanent unpacked tree.